ast/treemap/treemap.h

```diff
@@ -138,15 +138,21 @@ struct ReportedRubyException {
     core::LocOffsets onLoc;
 };
 
+enum class MapKind {
```

Comment:
Maybe `TreeMapKind`? This is currently `ast::MapKind` which is maybe a little unclear from context what it refers to.